### PR TITLE
Upgrade library and demo application to Angular 8

### PR DESCRIPTION
**Pull request by zbennett10**
@eitanfr 

I ensured that the following NPM commands still run and that all tests are passing:

demo:start
demo:build
demo:server
lib:start
lib:build

### CODE REVIEW

**Comment by zbennett10** [comment]
Note: The circleci fails because Angular 8 needs at least Node 10 to build correctly.

**Comment by eitanfr** [comment]
Cool thx!!! 
I change the CI config

File: projects/angular-cesium/src/lib/angular-cesium-widgets/components/polygons-editor/polygons-editor.component.ts [hunk 1]
```diff
@@ -99,9 +99,15 @@ export class PolygonsEditorComponent implements OnDestroy {
   public editPolylines$ = new Subject<AcNotification>();
   public editPolygons$ = new Subject<AcNotification>();
 
-  @ViewChild('editPolygonsLayer') private editPolygonsLayer: AcLayerComponent;
-  @ViewChild('editPointsLayer') private editPointsLayer: AcLayerComponent;
-  @ViewChild('editPolylinesLayer') private editPolylinesLayer: AcLayerComponent;
+  public appearance = new Cesium.PerInstanceColorAppearance({ flat: true });
```
Comment (eitanfr) [inline projects/angular-cesium/src/lib/angular-cesium-widgets/components/polygons-editor/polygons-editor.component.ts]: please remove:
```
  public appearance = new Cesium.PerInstanceColorAppearance({ flat: true });
// ...
  public Number = Number;
```

Comment (zbennett10) [reply projects/angular-cesium/src/lib/angular-cesium-widgets/components/polygons-editor/polygons-editor.component.ts]: OK - I'll work on these changes today. Thanks

**Comment by zbennett10** [comment]
The rebase wasn't super clean as it didn't handle some of the renaming that happened but I think I got it all cleaned up! Let me know if there is anything else @eitanfr .

Thanks!

**Comment by eitanfr** [comment]
Thx a lot I will go over it.
I will publish the new version soon

**Pull request merged**